### PR TITLE
VMAAS-287 CVSS Base Score options labels updated

### PR DESCRIPTION
**Pull request by janelznic**

### CODE REVIEW

File: src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js [hunk 1]
```diff
@@ -13,9 +13,9 @@ import { addNotification } from '../../../Notifications';
 
 const CVSSOptions = [
     { value: 'all', label: 'All', disabled: false, from: '', to: '' },
-    { value: '0to3', label: 'Lower than 3', disabled: false, from: '', to: 3 },
-    { value: '3to7', label: '3 - 7', disabled: false, from: 3, to: 7 },
-    { value: '7to10', label: '7 - 10', disabled: false, from: 7, to: 10 }
+    { value: '0to3', label: 'CVSS < 3.0', disabled: false, from: '', to: 3 },
```
Comment (jiridostal) [inline src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js]: 3 is both in the first and second interval. It should be in the second one only.

File: src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js [hunk 2]
```diff
@@ -13,9 +13,9 @@ import { addNotification } from '../../../Notifications';
...
+    { value: '3to6', label: 'CVSS >= 3.0 and CVSS < 7.0', disabled: false, from: 3, to: 6 },
```
Comment (jiridostal) [inline src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js]: Let's include values between 6 and 7 as well. Max value from range 3-7 is 6.999 as we store the score in the format of `5.000`

**Comment by karelhala** [comment]
Let's wait for master merge and I'll merge it!

**Pull request merged**